docs/csharp/language-reference/compiler-messages/cs0106.md

```diff
@@ -36,11 +36,15 @@ The modifier 'modifier' is not valid for this item
  A class or interface member was marked with an invalid access modifier. The following examples describe some of these invalid modifiers:  
   
 -   The [static](../../../csharp/language-reference/keywords/static.md) and [public](../../../csharp/language-reference/keywords/public.md) modifiers are not permitted on interface methods.  
-  
+
+- The [static](../../../csharp/language-reference/keywords/static.md) modifier is not permitted on a local function. Whether a local function is static or instance is defined by its containing member.
```

Comment:
> Whether a local function is static or instance is defined by its containing member.

If a local function inside an instance method does not close over `this`, then it's generated as a `static` method. What does actually define whether a local function is static or instance? Whether it *could* access `this`? Or what IL is generated for it?

So, maybe just remove this sentence altogether? I'm not sure it's actually useful.